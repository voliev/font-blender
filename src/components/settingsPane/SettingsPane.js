import React from "react";
import TypeScaleContainer from "./TypeScaleContainer";
import Elements from "./Elements";
import Background from "./Background";
import SettingsPaneFooter from "./SettingsPaneFooter";
import "./styles/SettingsPane.css";

const SettingsPane = () => (
  <div className="SettingsPane">
    <TypeScaleContainer />
    <Elements />
    <Background />
    <SettingsPaneFooter />
  </div>
);

export default SettingsPane;
