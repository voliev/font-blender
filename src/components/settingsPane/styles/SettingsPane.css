.SettingsPane {
  background-color: var(--off-white);
  min-height: calc(100vh - 4rem);
  overflow-y: auto;
  -webkit-overflow-scrolling: touch;
  width: var(--side-pane-width);
}
