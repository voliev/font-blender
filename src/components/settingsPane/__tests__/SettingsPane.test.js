import React from "react";
import { shallow } from "enzyme";
import SettingsPane from "../SettingsPane";

it("should render without crashing", () => {
  const wrapper = shallow(<SettingsPane />);

  expect(wrapper).toMatchSnapshot();
});
