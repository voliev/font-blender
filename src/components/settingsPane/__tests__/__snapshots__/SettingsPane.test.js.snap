// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<div
  className="SettingsPane"
>
  <Connect(TypeScale) />
  <Connect(Elements) />
  <Connect(Background) />
  <Connect(SettingsPaneFooter) />
</div>
`;
